ui/components/ui/textarea/README.mdx

```diff
@@ -0,0 +1,15 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import TextArea from '.';
+
+# TextArea
+
+TextAreas communicate actions that users can take.
```

Comment:
Think this is left over from the copied Button docs page should be something like:
```suggestion
TextArea allows users to enter text into the UI
```